03.bowling/bowling.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def make_frames(shots)
+  frames = []
+  shots.each_slice(2) do |s|
+    frames << if s[0] == 10
+                [s[0]]
+              else
+                s
+              end
+  end
+  frames
+end
+
+def cal_point(frames)
+  point = 0
+  frames.each_with_index do |frame, index|
+    break if index >= 10
```

Comment:
```suggestion
  frames.take(10).each_with_index.sum do |frame, index|
```

takeメソッド + sumメソッドにするとpoint変数を無くせるかも？